docs/content/docs/data-structures/overview.md

```diff
@@ -112,6 +113,7 @@ Typical scenarios require the connected clients to "agree" on some course of act
 - Upgrade a data schema. (All clients agree to simultaneously make the change.)
 
 <!-- Links -->
-[SharedMap]: https://fluidframework.com/docs/apis/map/sharedmap/
+[SharedCounter]: {{< relref "counter.md" >}}
 [SharedDirectory]: https://fluidframework.com/docs/apis/map/shareddirectory/
```

Comment:
@Rick-Kirkham @tylerbutler it seems to me like the links here should point to the overview articles, rather than directly to the API docs. The overview articles should link to the API docs as appropriate (in my opinion), and we should funnel readers to the articles instead here. What are your thoughts? I can update these while I'm here if we're in agreement on this.

Reply:
Which overview articles did you have in mind? I think that our intention is that readers aren't here in the Data Structures section until they are passed the beginner Fluid dev stage.

Reply:
Like the SharedCounter article I'm adding here and the existing [SharedMap](https://fluidframework.com/docs/data-structures/map), etc. articles in this directory. They're in the same dir as this doc, so I don't think the intended flow is broken. And I think linking to the DDS API docs before we give a higher-level overview of what they do is probably more confusing than useful.

Reply:
I see. Then I think you have the right idea about the links.